modules/messages/src/lib.rs

```diff
@@ -810,13 +809,19 @@ fn send_message<T: Config<I>, I: 'static>(
 > {
 	ensure_normal_operating_mode::<T, I>()?;
```

Comment:
I think we can move `ensure_normal_operating_mode::<T, I>()?;` under the `send_message` dispatch call  and reuse `ensure_not_halted()` to do the same thing. In this way, `ensure_normal_operating_mode` can be deleted.

Reply:
Yep. But keep this for now.